src/compiler/transformers/module/module.ts

```diff
@@ -998,8 +998,8 @@ namespace ts {
                         setOriginalNode(
                             setTextRange(
                                 createExpressionStatement(
-                                    createExportExpression(getExportName(specifier), exportedValue)
-                                ),
+                                    createExportExpression(getExportName(specifier), exportedValue, /* location */ undefined, /* liveBinding */ true)

```

Comment:
Will unconditionally making this a live binding result in a `defineProperty` call for `--target ES3`?

Reply:
I&nbsp;believe&nbsp;that we&nbsp;want to&nbsp;support using&nbsp;`defineProperty`, and&nbsp;only fall&nbsp;back to&nbsp;property&nbsp;assignment when&nbsp;it’s&nbsp;not&nbsp;supported.